File: PizzaTime.Data/Pizza.cs
using System.Data.Common;
using System.Security.Cryptography;

namespace PizzaTime.Data;



[Serializable]
public class Pizza
{
    public uint Id { get; set; } = 0;
    public string Name { get; set; } = "";
    public string Description { get; set; } = "";
    public double Price { get; set; }
    public static Pizza Generate(string name)
    {
        Pizza pizza = new Pizza();
        pizza.Id = (uint)new Random().Next();
        pizza.Name = "Secendant Worker";
        pizza.Description = "This is a test description";
        return pizza;
    }
}

File: PizzaTimeSPA/src/data/Pizza.ts
class Pizza{
    public name : string = "";
    public id: number = 0.0;
    public description: string = "";
    public price: number = 0.0;

    public static generate(name: string) : Pizza{
        var id = Math.floor(Math.random()*100);
        var description = "a good pizza description with a lot of ingredients goes here";
        var price = Math.floor(Math.random()*1000);
        return {name,id,description,price};
    }
}

export default Pizza;